portal/views/user.py

```diff
@@ -226,7 +226,7 @@ def account():
         try:
             org_list = [Organization.query.filter_by(
                 id=org['organization_id']).one()
-                    for org in request.json['organizations']]
+                        for org in request.json['organizations']]
```

Comment:
This should probably be at the least indentation level possible (it's not part of the variable that will get appended to the list)